app/screens/channel_peek/channel_peek.js

```diff
@@ -61,8 +54,16 @@ export default class ChannelPeek extends PureComponent {
         }
     }
 
-    getVisiblePostIds = (props) => {
-        return props.postIds?.slice(0, 15) || [];
+    getVisiblePostIds = (updateState = true) => {
+        const visiblePostIds = this.props.postIds?.slice(0, 15) || [];
```

Comment:
For this one in particular, couldn't we just slice them in the render method instead of storing the sliced IDs in the state?

Reply:
Yeah in a few of them I believe we can prevent the double rendering from happening. Will do that sometime next week unless you want to take over the PR